### PR TITLE
Marcel/5.3/cmake libcarla image support list

### PR DESCRIPTION
**Pull request by MarcelPiNacy**
This branch changes how image formats are added to libcarla. Currently the user must change CMAKE_CXX_FLAGS which may lead to problems. This adds the variable LIBCARLA_IMAGE_SUPPORTED_FORMATS, which expects a semicolon separated list, like "png;tiff;jpeg".

### CODE REVIEW

File: CMake/Options/Common.cmake [hunk 1]
```diff
@@ -91,3 +91,9 @@ option (
   "Whether to convert build warnings to errors."
   OFF
 )
+
+carla_string_option (
+  LIBCARLA_IMAGE_SUPPORTED_FORMATS
+  "Semicolon-separated list of supported image formats by LibCarla."
```
Comment (xavisolesoft) [inline CMake/Options/Common.cmake:97]:
```suggestion
  "Semicolon-separated list of supported image formats by LibCarla. Available formats: png, jpeg, tiff"
```

**Review by xavisolesoft — CHANGES_REQUESTED** [review]

File: Unreal/CMakeLists.txt [hunk 2]
```diff
@@ -194,8 +194,10 @@ if (BUILD_PYTHON_API)
   )
 endif ()
 
+
+
 add_custom_target (
-  launch
+  launch-only
```
Comment (xavisolesoft) [inline Unreal/CMakeLists.txt:200]: Are this changes the same than this other review?

https://github.com/carla-simulator/carla/pull/7250/files

Comment (MarcelPiNacy) [reply Unreal/CMakeLists.txt]: Fixed, yeah this got mixed in.

File: CMake/Options/Common.cmake [hunk 3]
```diff
@@ -91,3 +91,9 @@ option (
...
+  "png"
+)
```
[the remark this answers is not in the document]
Comment (MarcelPiNacy) [reply CMake/Options/Common.cmake]: Fixed.

**Review by xavisolesoft — APPROVED** [review]

**Pull request merged**